docs/sources/static/configuration/create-config-file.md

```diff
@@ -42,7 +42,7 @@ integrations:
 ```
 
 In this example, we first must configure the `wal_directory` which is used to
-store metrics in a Write-Ahead Log. This is required, but ensures that samples
+store metrics in a Write-Ahead Log. This is not required, but ensures that samples
 will be redelivered in case of failure (e.g., network issues, machine reboot). We
```

Comment:
```suggestion
will be redelivered in case of failures, for example if there are network connectivity issues. We
```

@mattdurham would this suggestion cover the point you raised about reboots/restarts not handled by WAL?

Reply:
Yes, that is primary use case.